com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/UpdateAppStep.java

```diff
@@ -219,12 +219,13 @@ private boolean hasChanged(StagingExtended staging, Staging existingStaging) {
         String stack = staging.getStack();
         Integer healthCheckTimeout = staging.getHealthCheckTimeout();
         String healthCheckType = staging.getHealthCheckType();
+        String healthCheckHttpEndpoint = staging.getHealthCheckHttpEndpoint();
         Boolean sshEnabled = staging.isSshEnabled();
         return (buildpackUrl != null && !buildpackUrl.equals(existingStaging.getBuildpackUrl()))
             || (command != null && !command.equals(existingStaging.getCommand()))
             || (stack != null && !stack.equals(existingStaging.getStack()))
             || (healthCheckTimeout != null && !healthCheckTimeout.equals(existingStaging.getHealthCheckTimeout()))
-            || (healthCheckType != null) || (sshEnabled != null);

```

Comment:
shouldn't these three null cheks be transformed to !<new-staging-parameter>.equals(<existing-staging-parameter>)? :)

Reply:
Currently not, because now they are part of StagingExtended but soon a fix for this will be provided where they will be part of Staging and this check will be implemented :)

Reply:
aha. I thought that this is the refactoring